packages/react/src/components/List/ListItem.tsx

```diff
@@ -178,15 +178,15 @@ class ListItem extends UIComponent<ReactProps<ListItemProps>, ListItemState> {
       },
     })
 
-    const hasHeaderPart = headerElement || headerMediaElement
+    const hasHeaderPart = !!(headerElement || headerMediaElement)
```

Comment:
Fixes:
```
Warning: Failed prop type: Invalid prop `column` of type `object` supplied to `Flex`, expected `boolean`.
    in Flex (created by ListItem)
    in ListItem (created by List)
    in ul (created by List)
    in List (created by ListExampleSelectable)
```

Reply:
But why is it in this PR? Shouldn't it go to its own with a changelog entry?

Reply:
Will create a separate PR for it

Reply:
See #1266.